kong/plugins/aws-lambda/schema.lua

```diff
@@ -0,0 +1,62 @@
+local function check_invocation_type(given_value, given_config, col, tbl, schema, opts)
+	if given_value ~= "RequestResponse" then
+		return false, "not supported"
+	end
+end
+
+local function check_log_type(given_value, given_config)
+	if given_value ~= "None" then
+		return false, "not supported"
+	end
+end
+
+local function check_qualifier(given_value, given_config)
+	if given_value ~= nil then
+		return false, "not supported"
+	end
+end
+
+return {
+	no_consumer = true,
+	fields = {
+		aws_region = {
+			type = "string",
+			required = true },
+		function_name = {
+			type = "string",
+			required = true },
+		qualifier = {
+			type = "string",
+			required = false,
+			--default = "$LATEST",
```

Comment:
I actually need to comment out qualifier altogether as it is not yet supported in the signing code. I think I can add this, but it might take a little time. I don't think this is MVP, though. But this definitely should be commented out altogether until it is supported.